### PR TITLE
Fix  an issue in the helper functions

### PR DESCRIPTION
**Pull request by pipo02mix**
The problem appears when webhook was enabled. Port number of the endpoint was converted to a string wrongly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [v0.8.2] - 2020-06-17
+
+- Fix template helpers issue converting port number for the webhook endpoint.
```
Comment (sslavic) [inline CHANGELOG.md:12]:
```suggestion
### Fixed

- Fix template helpers issue converting port number for the webhook endpoint.
```

Last two releases and this one being prepared introduced slight inconsistency with the format/structure of the change log - we should use these listing headings as in https://keepachangelog.com/en/1.0.0/

We can fix the other two in a follow-up PR.

File: helm/kong-app/templates/_helpers.tpl [hunk 2]
```diff
@@ -190,7 +190,7 @@ The name of the service used for the ingress controller's validation webhook
 {{- $_ := set $autoEnv "CONTROLLER_ELECTION_ID" (printf "kong-ingress-controller-leader-%s" .Values.ingressController.ingressClass) -}}
 {{- $_ := set $autoEnv "CONTROLLER_KONG_URL" (include "kong.adminLocalURL" .) -}}
 {{- if .Values.ingressController.admissionWebhook.enabled }}
-  {{- $_ := set $autoEnv "CONTROLLER_ADMISSION_WEBHOOK_LISTEN" (printf "0.0.0.0:%d" .Values.ingressController.admissionWebhook.port) -}}
+  {{- $_ := set $autoEnv "CONTROLLER_ADMISSION_WEBHOOK_LISTEN" (printf "0.0.0.0:%s" .Values.ingressController.admissionWebhook.port) -}}
```
Comment (sslavic) [inline helm/kong-app/templates/_helpers.tpl:193]: Issue is present in upstream chart https://github.com/Kong/charts/blob/master/charts/kong/templates/_helpers.tpl#L193

Yasser was putting lots of effort to minimize customizations inventory, try to push fixes like this to the upstream (eventually?). He was mentioning having some automation to sync with the upstream changes, while still making it possible to have some customizations in our "fork". Not sure about the process details but curious to learn.

Will check readme if process was documented, or can derive it from history. @ced0ps do you maybe know already how Yasser was doing it? 

//cc @yasn77

Comment (pipo02mix) [reply helm/kong-app/templates/_helpers.tpl:193]: ok let's unblock our customer and I do a PR to upstream now

Comment (yasn77) [reply helm/kong-app/templates/_helpers.tpl:193]: Yep... @pipo02mix could you also create a PR upstream? 

The way I am keeping things in sync is using https://gist.github.com/yasn77/cb0600bebb9237fbd73d7ca67b4ff211. That downloads the tarballs and then does a cherry-pick of commits. But if the change already exists in this fork, then it's just ignored. So it's fine to make a change here first (so out customers are ok) and later pull the change from upstream.

**Comment by yasn77** [comment]
This PR fixes https://github.com/giantswarm/adidas/issues/560

**Pull request merged**